_posts/2014-07-31-dev-meetup-july-26-2014.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Monthly Dev Meetup July 2014"
+date: 2014-07-31 19:37
+comments: true
+author: "Saroj Maharjan"
+categories: [activities]
+author-twitter-name: "zoras"
+author-github-name: "zoras"
+---
+
+The tech scene in Kathmandu is ever growing but it is rather lacking a sense of community. Dev Meetup aims to fill this void with a regular meetup for developers where like minded individuals can gather, get to know each other and share their ideas/views.
+
+The meetup will be carried out monthly (last Saturday of the month) to cover web development topics like 'ruby', 'rails', 'javascript', 'devops', 'databases', etc.
+
+The first successful [Dev Meetup](https://www.facebook.com/events/768447783198796/) was held on July 26, 2014 at CloudFactory, the next Dev Meetup is scheduled for August 30th, 2014 at the same venue.
+
+Sign up at https://plus.google.com/events/cr78vpsi1466vv0ldj990fdgdr8 to join us. Kudos to CloudFactory for sponsoring both the events.
+
+Follow us [@ruby_nepal](http://twitter.com/ruby_nepal) for latest updates and news.
+
+If you are not sure what to expect or missed the 1st Dev Meetup, check out the pics and videos from the first one.
+
+** Pictures **
+
+![](https://pbs.twimg.com/media/BtdOPL3CMAAlPlN.jpg)
+![](https://pbs.twimg.com/media/BtdmzUJCUAEDi0W.jpg)
+
+** FB Event Pics and Details: ** 
+https://www.facebook.com/events/768447783198796/
+
+** Videos And Slides:  **
+
+_I) 10 Ways We Can Be Better by Mark Sears_ - [@marktsears](http://twitter.com/marktsears)
+
+![](https://d3j5vwomefv46c.cloudfront.net/photos/large/861728292.jpg?1406529728)
+
+[comment]: <> ( youtube COBCnZls5rE )
```

Comment:
@zoras  dai, ```youtube COBCnZls5rE``` This one is not working in jekyll so I added comment here. What should do for it ? Any suggestions ?

Reply:
Previously we were displaying youtube videos using plugins
1. https://github.com/RubyNepal/rubynepal.github.io/blob/source/_plugins/youtube.rb
2. https://github.com/RubyNepal/rubynepal.github.io/blob/source/_plugins/video_tag.rb

Now we can use a single plugin gem https://github.com/dommmel/jekyll-youtube#installation

I'll work on adding it tomorrow.